src/app/components/Curation/CurationGrid/index.tsx

```diff
@@ -1,24 +1,43 @@
 /** @jsx jsx */
 import { jsx } from '@emotion/react';
+import { VISUAL_PROMINENCE, Summary } from '#app/models/types/curationData';
+import extractWorldServiceFromUrl from '#app/lib/utilities/extractWorldServiceFromUrl';
+import { ServiceContextProvider } from '#app/contexts/ServiceContext';
 import moment from 'moment';
+import { use } from 'react';
+import { RequestContext } from '#app/contexts/RequestContext';
 import styles from './index.styles';
 import CurationPromo from '../CurationPromo';
+import HighImpactPromo from '../HighImpactPromo';
 import { CurationGridProps } from '../types';
-import { Summary } from '../../../models/types/curationData';
+
+const isMediaType = (summary: Summary): boolean => {
```

Comment:
This is a bit more readable
```suggestion
const isMedia = (summary: Summary): boolean => {
```